app/Http/Controllers/AcceptancesController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Acceptance;
+use App\Submission;
+use Illuminate\Http\Request;
+
+class AcceptancesController extends Controller
+{
+    public function store(Request $request)
+    {
+        $submission = Submission::findOrFail($request->input('submissionId'));
+        if (auth()->user()->id != $submission->talkRevision->talk->author_id) {
+            return response('', 401);
```

Comment:
Is this better than `abort(401)` in any way? I honestly don't know :)

Reply:
When I was working on it `abort(401)` didn't return a 401 API response like I expected, it was throwing a 500 error, changing to this stopped that.

Reply:
Weird. That shouldn't happen but we can test later.